openapi.yaml

```diff
@@ -1301,8 +1301,8 @@ components:
             description: Average (arithmetic mean) score of the advisory aggregated from *all* related vulnerability assertions.
             nullable: true
           average_severity:
-            type: string
-            description: Average (arithmetic mean) severity of the advisory aggregated from *all* related vulnerability assertions.
+            allOf:
+            - $ref: '#/components/schemas/Severity'
```

Comment:
It is surprising that changing the type results in the field description being dropped. Does anyone know why this might be happening?